api/api/accounts/managers.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.base_user import BaseUserManager
+
+
+class CustomUserManager(BaseUserManager):
+
+    def create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError('The Email must be set')
+        if not password:
+            raise ValueError("User must have a password")
```

Comment:
Throwing this exception here means we are forcing users to have passwords. As we are using OAuth2 for now. That means we will not use passwords and so this check can be skipped.